assemblies/assembly_about-rhdh.adoc

```diff
@@ -0,0 +1,28 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-10-07
+
+ifdef::context[:parent-context-of-about-rhdh: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+ifndef::context[]
+[id="about-rhdh"]
+endif::[]
+ifdef::context[]
+[id="about-rhdh_{context}"]
+endif::[]
+= About rhdh
+
+:context: about-rhdh
+
+{product} is a fully supported, enterprise-grade, open developer platform that you can use to build developer portals. This platform contains a supported and opinionated framework that helps reduce the friction and frustration of developers while boosting productivity. {product} simplifies decision-making by providing a developer experience that presents a selection of internally approved tools, programming languages, and developer resources within a self-managed portal. As a developer, you can use {product} to experience a streamlined development environment. {product} is driven by a centralized software catalog, providing efficiency to your microservices and infrastructure. It enables your product team to deliver quality code without any compromises.
+
+
+
+include::assembly_benefits-of-rhdh.adoc[leveloffset=+1]
+
+
+include::assembly_sizing-requirements-for-rhdh.adoc[leveloffset=+1]
```

Comment:
These two modules are not assemblies. Consider using the accurate module type (concept, procedure, or reference).